<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeEditorDialog</class>
 <widget class="QDialog" name="RecipeEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lblRecipeName">
         <property name="text">
          <string>Recipe Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineRecipeName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblRecipeInstruction">
         <property name="text">
          <string>Recipe Instruction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtRecipeInstruction"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblIngredients">
         <property name="text">
          <string>Ingredients</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listIngredients"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblIngredient">
         <property name="text">
          <string>Ingredient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineIngredientName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineIngredientAmount"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineIngredientMeasurement"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddIngredient">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteIngredient">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecipeEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecipeEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
